ddtelemetry-ffi/cbindgen.toml

```diff
@@ -15,9 +15,15 @@ includes = ["datadog/common.h"]
 
 [export]
 prefix = "ddog_"
+renaming_overrides_prefixing = true
 
 [export.mangle]
-rename_types="SnakeCase"
+rename_types = "PascalCase"
+
+[export.rename]
+"ParseTagsResult" = "ddog_Vec_Tag_ParseResult"
+"PushTagResult" = "ddog_Vec_Tag_PushResult"
+"FILE" = "FILE"
```

Comment:
There's a few more renames missing (there's quite a few telemetry-specific types that are being placed in the common namespace), but again this was already the case so it's more of an FYI ;)